### DISCUSSION TITLE
Media file problem when migrate to new S3 config (Ceph S3 to AWS S3)

### DISCUSSION THREAD

**Discussion by nh4ttruong** [outline/outline#7353] [Self Hosting] [answered]
I migrated the Ceph S3 configurations to AWS S3. However, the media files' paths (such as jpg, png...) were not updated to the AWS S3 path and they are still using the old configuration (Ceph S3 bucket). The document media was not affected by this change, and all documents still show the media path in the new configuration correctly (AWS S3).

Old config url still appeared:
![image](https://github.com/user-attachments/assets/754340c5-d1c1-4dff-b4d2-f6763ef11a89)

Document media still show:
![image](https://github.com/user-attachments/assets/99d25dba-c95b-4adb-b206-999ba2b41349)

Note: I known that Ceph S3 is not in S3 compatiable list, but why did document media show correctly while avatar, detail image not?

**To Reproduce**
1. Migrate old S3 bucket (Ceph S3) to new S3 (AWS S3)
2. Change config variable in Outline
3. Restart service

**Expected behavior**
Avatars, detail images... are shown correctly with new path S3 config

**Outline (please complete the following information):**
- Install: self hosted
- Version: v0.78.0

**Comment by tommoor** [comment, marked as answer]
> Avatars, detail images... are shown correctly with new path S3 config

These values are stored in the database, you would need to update them manually with a query in this case. The locations you're looking for are `teams.avatarUrl` and `users.avatarUrl`

**Comment by nh4ttruong** [comment]
Thank you so much @tommoor. I have a question that why documents are correctly path?
